README.md

```diff
@@ -7,10 +7,11 @@ The ringbuffer crate provides safe fixed size circular buffers (ringbuffers) in
 
 Implementations for three kinds of ringbuffers, with a mostly similar API are provided:
 
-| type | description |
-| --- | --- |
-| `AllocRingBuffer` | Ringbuffer allocated on the heap at runtime. This ringbuffer is still fixed size and requires alloc. |
-| `ConstGenericRingBuffer` | Ringbuffer which uses const generics to allocate on the stack. |
+| type                      | description                                                                                                                                                                      |
+|---------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `AllocRingBuffer`         | Ringbuffer allocated on the heap at runtime. This ringbuffer is still fixed size. This requires alloc and the alloc feature.                                                     |
```

Comment:
```suggestion
| `AllocRingBuffer`         | Ringbuffer allocated on the heap at runtime. This ringbuffer is still fixed size. This requires the alloc feature.                                                |
```

Reply:
I know you mean the difference between having an allocator and the feature `alloc` but it is just a bit confusingly worded.

Alternatively you can also say ".. requres an allocator and consequently the alloc feature" or so